.gitignore

```diff
@@ -13,3 +13,6 @@ packages
 # Include when developing application packages.
 pubspec.lock
 .packages
+
+# Don't commit IntelliJ project directories
+.idea
```

Comment:
Done. How do you choose what should be ignored globally and what should be ignored locally?